From 068b617eac75aa14b1507edb297c2dbd02953fff Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Tue, 13 Apr 2021 09:44:11 -0500
Subject: [PATCH 081/196] Update chainparamsconstants for testnet (#147)

This allows other nodes to get out of initial block download.
We also remove the default assumed valid block for now.
---
 src/networks/abc/chainparamsconstants.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/networks/abc/chainparamsconstants.cpp b/src/networks/abc/chainparamsconstants.cpp
index 877ba72f3..e5e87d658 100644
--- a/src/networks/abc/chainparamsconstants.cpp
+++ b/src/networks/abc/chainparamsconstants.cpp
@@ -10,9 +10,9 @@ namespace ChainParamsConstants {
     const uint64_t MAINNET_ASSUMED_BLOCKCHAIN_SIZE = 210;
     const uint64_t MAINNET_ASSUMED_CHAINSTATE_SIZE = 3;
 
-    const BlockHash TESTNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("000000000b94431d3622a1d05fc092a9859a898c1d866e8b616483de212c3a4a");
-    const uint256 TESTNET_MINIMUM_CHAIN_WORK = uint256S("00000000000000000000000000000000000000000000006e80379ea431ff95c0");
-    const uint64_t TESTNET_ASSUMED_BLOCKCHAIN_SIZE = 55;
+    const BlockHash TESTNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("0000000000000000000000000000000000000000000000000000000000000000");
+    const uint256 TESTNET_MINIMUM_CHAIN_WORK = uint256S("000000000000000000000000000000000000000000000000000010fb52db234e");
+    const uint64_t TESTNET_ASSUMED_BLOCKCHAIN_SIZE = 2;
     const uint64_t TESTNET_ASSUMED_CHAINSTATE_SIZE = 2;
 } // namespace ChainParamsConstants
 
-- 
2.39.5 (Apple Git-154)

